From 1c61b9001cf2053df9d05fa518b2c2a9be99f903 Mon Sep 17 00:00:00 2001
From: Gary Lin <glin@suse.com>
Date: Thu, 1 Aug 2019 10:38:51 +0800
Subject: [PATCH] Initialize compress variable

$compress in pesign-gen-repackage-spec wasn't initialized and this may
caused a warning like this:

Use of uninitialized value $compress in string eq at /usr/lib/rpm/pesign/pesign-gen-repackage-spec line 422.

Signed-off-by: Gary Lin <glin@suse.com>
---
 pesign-gen-repackage-spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pesign-gen-repackage-spec b/pesign-gen-repackage-spec
index fef0a9d..fa0935e 100755
--- a/pesign-gen-repackage-spec
+++ b/pesign-gen-repackage-spec
@@ -30,7 +30,7 @@ my $directory;
 my $output = ".";
 my $cert_subpackage;
 my $kmp_basename;
-my $compress;
+my $compress = "";
 my @rpms;
 
 $ENV{LC_ALL} = "en_US.UTF-8";
-- 
2.22.0

